cmd/gce-pd-csi-driver/main.go

```diff
@@ -57,6 +57,8 @@ var (
 
 	maxprocs = flag.Int("maxprocs", 1, "GOMAXPROCS override")
 
+	errorBackoffInitialDuration = 200 * time.Millisecond
```

Comment:
Please make this a flag for errorBackoffMaxDuration and errorBackoffInitialDuration, with default values

Reply:
Good idea! Done

Reply:
can you also make the errorBackoffMaxDuration configurable through the flag

Reply:
Woops sorry I missed that, added errorBackoffMaxDuration. Note I did it in ms to match the initial duration flag, but lmk if you think I should change to minutes/seconds.